libs/blocks/merch/merch.js

```diff
@@ -122,11 +122,14 @@ export async function buildCta(el, params) {
   const context = await getCheckoutContext(el, params);
   if (!context) return null;
   const service = await initService();
+  const isCta = el.textContent?.startsWith('CTA');
```

Comment:
Why not reuse from line 144 `const isCta = searchParams.get('type') === 'checkoutUrl';`

Reply:
Because in both cases, we will have the same type as the checkout URL.